demo/index.html

```diff
@@ -50,13 +75,19 @@ <h3>Basic static grid</h3>
                 <tile col="2" row="1" height="1" width="1"></tile>
                 <tile col="3" row="1" height="2" width="1"></tile>
                 <tile col="4" row="0" height="3" width="2"></tile>
-                <div placeholder></div>
+                <div placeholder=""></div>
```

Comment:
There is no way we can declare an attribute without value ? The syntax is is bit ugly...